examples/kafka-hub/hub/modules/connections/connections.bal

```diff
@@ -17,34 +17,19 @@
 import ballerinax/kafka;
 import kafkaHub.config;
 
-final kafka:SecureSocket & readonly secureSocketConfig = {
-    cert: "./resources/brokercerts/broker.public.crt",
-    protocol: {
-        name: kafka:SSL
-    },
-    'key: {
-        certFile: "./resources/brokercerts/client.public.crt",
-        keyFile: "./resources/brokercerts/client.private.key"
-    }
-};
-
 // Producer which persist the current in-memory state of the Hub 
 kafka:ProducerConfiguration statePersistConfig = {
     clientId: "state-persist",
     acks: "1",
-    retryCount: 3,
-    secureSocket: secureSocketConfig,
-    securityProtocol: kafka:PROTOCOL_SSL
+    retryCount: 3
 };
 public final kafka:Producer statePersistProducer = check new (config:KAFKA_BOOTSTRAP_NODE, statePersistConfig);
 
 // Consumer which reads the persisted subscriber details
 kafka:ConsumerConfiguration websubEventsConsumerConfig = {
     groupId: "websub-events-receiver-" + config:CONSTRUCTED_SERVER_ID,
     offsetReset: "earliest",
-    topics: [ config:WEBSUB_EVENTS_TOPIC ],
-    secureSocket: secureSocketConfig,
-    securityProtocol: kafka:PROTOCOL_SSL
+    topics: [ config:WEBSUB_EVENTS_TOPIC ]
 };
 public final kafka:Consumer websubEventsConsumer = check new (config:KAFKA_BOOTSTRAP_NODE, websubEventsConsumerConfig);
```

Comment:
dont we need to change the `KAFKA_BOOTSTRAP_NODE` as well (to a plaintext endpoint)?

Reply:
Fixed with [0efd7d8ed517386edeb1c91d71c9e1bf690f3c46](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/882/commits/0efd7d8ed517386edeb1c91d71c9e1bf690f3c46)